src/network/networkd-link.c

```diff
@@ -2618,6 +2618,21 @@ static int link_set_ipv6_mtu(Link *link) {
         return 0;
 }
 
+static int link_is_static_address_configured(Link *link, Address *address) {
+        assert(link);
+        assert(address);
+
+        if (link->network) {
+                Address *net_address;
+                LIST_FOREACH(addresses, net_address, link->network->static_addresses) {
+                        if (address_equal(net_address, address))
+                                return 0;
+                }
```

Comment:
the {} are unnecessary here...

Reply:
Thanks, for the clarification. That's what I tried to figure with my question above since your coding style refers to single line block. For me this was rather multiple lines hence nests are counted differently here.